app/src/main/java/com/keylesspalace/tusky/util/ThemeUtils.kt

```diff
@@ -35,7 +35,8 @@ private const val THEME_DAY = "day"
 private const val THEME_BLACK = "black"
 private const val THEME_AUTO = "auto"
 private const val THEME_SYSTEM = "auto_system"
-const val APP_THEME_DEFAULT = THEME_NIGHT
+const val APP_THEME_DEFAULT = THEME_SYSTEM
+const val APP_THEME_DEFAULT_COMPAT = THEME_NIGHT
```

Comment:
I’m not sure of this one, the other idea I have is to set _one_ constant (`THEME_NIGHT`) from above as public, which is quite strange. Let me know what do you prefer.

Reply:
There's probably no harm in making all those constants public, which would then make the code in `TuskyApplication` a bit more explicit about exactly which value is being set.

Reply:
Done.